src/structures/Role.js

```diff
@@ -173,7 +173,7 @@ class Role extends Base {
    * @example
    * // Edit a role
    * role.edit({ name: 'new role' })
-   *   .then(updated => console.log(`Edited role ${updated.name} name to ${updated.name}`))
+   *   .then(updated => console.log(`Edited role name to ${updated.name}`))
```

Comment:
```suggestion
   *   .then(updated => console.log(`Updated role name to ${updated.name}`))
```

Reply:
I think it's fine to leave it as `Edited` because it's coming from the `edit` method.